phys/module_ra_rrtmg_lwk.F

```diff
@@ -9010,8 +9011,11 @@ subroutine cmbgb6
 !  old band 6:  820-980 cm-1 (low - h2o; high - nothing)
 !
 !-------------------------------------------------------------------------------
-   use parrrtm_k,   only : mg, nbndlw, ngptlw
-   use rrlw_kg06
+   use parrrtm_k,   only : mg, nbndlw, ngptlw, ng6
+   use rrlw_kg06_k, only : fracrefao, kao, kao_mco2, cfc11adjo, cfc12o,        &
```

Comment:
@weiwangncar @smileMchen 
Ming,
To get your GHG code to build, put in a small fix here. 
Note that the code you have says "rrlw_kg06", and it should say "rrlw_kg06_k". I put this error in a while ago.

Reply:
@davegill 
Dave,
I will change the code accordingly. Thanks.